include/state_machine.h

```diff
@@ -20,19 +20,20 @@ class StateMachine
 
   bool left();
   bool right();
-  bool up();
-  bool down();
+  bool shortcutPush();
+  bool shortcutDoublePush();
 
   bool back();
   bool select();
   bool activate();
 
 private:
-  void constructWalkMenu(State* from);
-  void constructSideStepMenu(State* from);
-  void constructStepMenu(State* from);
-  void constructSofaMenu(State* from);
-  void constructStairsMenu(State* from);
+  void constructObstacleMenu(State* from, State* walk, State* single_step, State* obstacles);
+  void constructWalkMenu(State* from, State* prev_gait);
+  // void constructSideStepMenu(State* from);
```

Comment:
Why has this been removed?

Reply:
Not exactly sure. But there is no side step menu so now I deleted it completely